tools/buildsteps/windows/win32-uwp/make-addons.bat

```diff
@@ -1,8 +1,6 @@
-@ECHO OFF

```

Comment:
line endings changed in whole file

Reply:
Not entirely sure how to fix that. I've since switched my checked out repo to autocrlf=true and cloning a new version without it checks it out as LF file endings.